### ISSUE TITLE
Display loss curves on Visdom

### ISSUE THREAD

**Issue by albusdemens** [facebookresearch/pytorch_GAN_zoo#99]
Hi, I'm training on an image dataset and I guess I should see the trend of the loss functions plotted in Visdom. I'm now at interaction 1700 and there's nothing displayed yet. Am I doing something wrong? I'm running the code in its standard configuration.

As commands, I gave the following:
```
python -m visdom.server &&
python train.py StyleGAN -c maps.json
```

**Comment by Molugan** [comment]
Hello,

Sorry for the huge delay.
Try launching visdom in another terminal before running the training.
